### PR TITLE
Do not override bricks via SC. PART 2

### LINKED ISSUE #2473: Do not override bricks via SC. PART 2

### PR DESCRIPTION
**Pull request by DenisVorop**
## PR includes

- [x] Refactor

## Related issues

Resolve #2473 

Part of:
- https://github.com/taskany-inc/issues/issues/2288

### CODE REVIEW

File: src/components/Badge/Badge.module.css
Comment (awinogradov) [inline src/components/Badge/Badge.module.css:1]: https://taskany-inc.github.io/bricks/?path=/story/harmony-badge--default

File: src/components/HistoryRecord/HistoryRecord.module.css [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+.IssueListItem a {
+    display: inline-flex;
+}
+
+.IssueListItem + .IssueListItem::before {
+    content: ',';
+    padding-right: 0.25em;
+}
+
+.Badge {
```
Comment (awinogradov) [inline src/components/HistoryRecord/HistoryRecord.module.css:84]: Looks like the new one?

File: src/components/HistoryRecord/HistoryRecord.module.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+.HistoryActivityFeedItem .Circle {
+    /*
+        no-magic: this negative margin needs for align
+        icon by center of first line in content
+    */
+    margin-top: -3px;
+}
+
+.FlexReset {
```
Comment (awinogradov) [inline src/components/HistoryRecord/HistoryRecord.module.css:42]: ?

File: src/components/HistoryRecord/HistoryRecord.module.css [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+}
+
+.Circle {
+    background-color: var(--background-color);
+}
+
+.HistoryActivityFeedItem .Circle {
+    /*
+        no-magic: this negative margin needs for align
+        icon by center of first line in content
```
Comment (awinogradov) [inline src/components/HistoryRecord/HistoryRecord.module.css:37]: @LamaEats added align for icons. Why this no-magic is here?

File: src/components/HistoryRecord/HistoryRecord.module.css [hunk 4]
```diff
@@ -0,0 +1,108 @@
+.HistoryRecordWrapper {
```
Comment (awinogradov) [inline src/components/HistoryRecord/HistoryRecord.module.css:1]: We have new HistoryRecord. Do we really need to add so many overrides. https://taskany-inc.github.io/bricks/?path=/story/harmony-historyrecord--default

File: src/components/InlineTrigger/InlineTrigger.module.css [hunk 5]
```diff
@@ -0,0 +1,28 @@
+.InlineTrigger {
```
Comment (awinogradov) [inline src/components/InlineTrigger/InlineTrigger.module.css:1]: Looks like Badge component

**Review by awinogradov — CHANGES_REQUESTED** [review]

**Review by tsumo — COMMENTED** [review]
Conflicts with main brach

**Comment by DenisVorop** [comment]
blocked by https://github.com/taskany-inc/issues/issues/2416

**Review by 9teen90nine — APPROVED** [review]

**Review by tsumo — APPROVED** [review]

**Pull request merged**